routes/api.php

```diff
@@ -9,6 +9,7 @@
     Route::post('/auth')->controller('AuthController')->action('login'),
     Route::post('/refresh-token')->controller('AuthController')->action('refresh'),
     Route::post('/users')->controller('AuthController')->action('signup'),
+    Route::post('/users/custom/register')->controller('AuthController')->action('signupByRegisterRole'),
```

Comment:
lest setup this to be /users/custom-registration or /users-custom-registration better